client/src/components/Questions/QuestionsList/Answer/Button/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './Button.css';
+
+function Button({ text, handleClick }) {
+  return (
+    <button
+      type="button"
+      onClick={handleClick}
+      className={style.button}
+    >
+      <span>{text}</span>
+    </button>
+  );
+}
+
+Button.propTypes = {
```

Comment:
Do you also need defaultProps here?

Reply:
It is marked 'isRequired' so there is no need for default

Reply:
It is marked 'isRequired' so no need for defaultProps